src/components/CCIP/Drawer/LaneDrawer.tsx

```diff
@@ -235,23 +280,68 @@ function LaneDrawer({
                         </td>
 
                         <td>
-                          {/* TODO: Fetch rate limits from API for both inbound and outbound
-                              GET /api/ccip/v1/lanes/by-internal-id/{source}/{destination}/supported-tokens?environment={environment}
-                              Response will contain both standard and custom rate limits per token */}
-                          Disabled
+                          {isLoadingRateLimits ? (
+                            "Loading..."
+                          ) : allLimits.standard ? (
+                            allLimits.standard.isEnabled ? (
+                              formatRateLimit(allLimits.standard.capacity)
+                            ) : (
+                              "Disabled"
+                            )
+                          ) : (
+                            <span style={{ display: "inline-flex", alignItems: "center", gap: "4px" }}>
+                              Unavailable
+                              <Tooltip
+                                label=""
+                                tip="Rate limit data is currently unavailable. You can find the Token Pool rate limit by reading the Token Pool contract directly on the relevant blockchain."
+                                style={{
+                                  display: "inline-block",
+                                  verticalAlign: "middle",
+                                }}
+                              />
+                            </span>
+                          )}
```

Comment:
All these conditions should not happen in the UI. Please abstruct it.. If you find yourself doign an if/ifsele/else, you should abstract. It is very hard to ready

Reply:
Same apply to all the i, if else present in this PR